kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/AbstractOperationDSL.kt

```diff
@@ -21,8 +22,8 @@ abstract class AbstractOperationDSL(
 
     private fun resolver(function: FunctionWrapper<*>): ResolverDSL {
         try {
-            require(function.hasReturnType()) {
-                "Resolver for '$name' has no return value"
+            if (!function.hasReturnType()) {
+                throw SchemaException("Resolver for '$name' has no return value")
             }
         } catch (e: Throwable) {
             if ("KotlinReflectionInternalError" !in e.toString()) {
```

Comment:
off-topic:

Do you have any idea why we'd want to just swallow specific exceptions?

Reply:
I was wondering the same and couldn't find an answer. Maybe java-interoperability?